test/functional/mempool_package_rbf.py

```diff
@@ -37,15 +38,7 @@ def set_test_params(self):
         ]] * self.num_nodes
 
     def assert_mempool_contents(self, expected=None):
-        """Assert that all transactions in expected are in the mempool,
-        and no additional ones exist.
-        """
-        if not expected:
-            expected = []
-        mempool = self.nodes[0].getrawmempool(verbose=False)
-        assert_equal(len(mempool), len(expected))
-        for tx in expected:
-            assert tx.rehash() in mempool
+        mempool_util.assert_mempool_contents(self, self.nodes[0], expected, sync=False)
```

Comment:
inlining `assert_mempool_contents` and removing dead code results in the exact same code as before 👍